### ISSUE TITLE
RuntimeError: open file failed

### ISSUE THREAD

**Issue by verculum** [alievk/avatarify#375]
**Describe the bug**

**To Reproduce**
Rename user, reinstall conda and avatarify, deleted .cache folder, сhanged permissions 

**Info (please complete the following information):**
 - OS: Windows 7 x64
 - GPU model: GT 210

I think it's the Cyrillic path of the folder .cache, but after deleting the script itself loads the file there, but then can not read. I want to run it on Google Colab

**Comment by JohanAR** [comment]
Yes, it could be caused by cyrillic letters in the path, and possibly incorrect handling of unicode characters. Might be able to fix it if the error is in avatarify, but the error could also be in pytorch in which case it would be more difficult to do something about. Unfortunately I can't reproduce the issue since I don't have Windows

Could you please copy and post the whole output from when you run run_windows.bat? If there's a stack trace I might be able to guess what needs to be changed

**Comment by zuo12138** [comment]
Have you worked out? I find the same problem now.
